redis.go

```diff
@@ -25,6 +26,10 @@ func SetLogger(logger *log.Logger) {
 }
 
 type baseClient struct {
+	//Use context to which carries deadlines, cancelation signals,
```

Comment:
- tambahin spasi sebelum Use
- ga pake `to`, lgsg aja `which carries`
- cancellation

Reply:
*cancellation itu alternative spelling bukannya?

`cancelation = the action of canceling something that has been arranged or planned.`
https://en.wiktionary.org/wiki/cancelation

Reply:
dari link di atas
![capture](https://user-images.githubusercontent.com/4661221/42866047-a86cd90c-8a95-11e8-998d-78bce93a610c.PNG)

cancelation is alternative spelling of cancellation. cancelation adalah ejaan alternatif dari cancellation. jadi yg alternatif itu cancelation.

better link oxford: https://en.oxforddictionaries.com/definition/cancellation

Reply:
oiya yah. kurang teliti saya.